dnf/base.py

```diff
@@ -1218,7 +1222,7 @@ def _translate_comps_pkg_types(pkg_types):
                 ret |= enum
         return ret
 
-    def group_install(self, grp, pkg_types, exclude=None):
+    def group_install(self, grp_id, pkg_types, exclude=None):
```

Comment:
Yeah, and don't forget to follow https://github.com/rpm-software-management/dnf/wiki/Hacking#compatibility-changes 8-) I mean, bump the minor version (if not bumped already and deprecate the old interface).